server/data/restClient.test.ts

```diff
@@ -24,15 +31,41 @@ describe('restClient', () => {
     nock.cleanAll()
   })
 
+  describe('get', () => {
+    it('should make a GET request', async () => {
+      fakeApprovedPremisesApi.get(`/some/path`).reply(200, { some: 'data' })
+
+      const result = await restClient.get({ path: '/some/path' })
+
+      expect(result).toEqual({ some: 'data' })
+      expect(nock.isDone()).toBeTruthy()
+    })
+
+    it('should omit the X-SERVICE-NAME header when the configuration does not include a service name', async () => {
+      config.apis.approvedPremises.serviceName = undefined
```

Comment:
When we stub our boundary here we lose a small bit coverage where the system extracts the environment variable and any potential default logic. 

In Rails we have a [gem](https://github.com/thoughtbot/climate_control) that lets us stub the environment variables. I'm not sure if that's possible or conventional in Typescript but just an optional idea incase it is.

Reply:
This actually pushed me in the other direction of removing the dependency on environment variables entirely from this test!

Particularly as it was modifying env vars, it felt like it was breaking the self-containedness of our test, and it isn't `restClient.test.ts`'s role to be testing env var parsing